app/views/admin/partners/_form.html.erb

```diff
@@ -6,8 +6,10 @@
 
   <%= f.input :name, class: "form-control" %>
   <%= f.input :slug, class: "form-control" if policy(@partner).permitted_attributes.include? :slug %>
-  <%= f.input :short_description, class: "form-control", label: 'Short Description', input_html: { rows: 7 } %>
 
+  <%= f.input :summary, class: "form-control", label: 'Summary' %>
```

Comment:
Restrict the max length of the field to 200 as well. If we can get the 200 value from the model even better. Or does it infer this automatically?